config/software/datadog-agent.rb

```diff
@@ -25,7 +31,18 @@
   mkdir  "#{install_dir}/agent/"
 
   # Agent code
+  mkdir "#{install_dir}/agent/checks.d"
   copy 'checks.d', "#{install_dir}/agent/"
+  checks = Dir["#{integrations_dir}/*"]
+  if ENV["USE_INTEGRATION_SDK"]
```

Comment:
What do you think of checking the existence of `integrations_dir` instead? I was thinking that this could also protect us from errors if the directory doesn't exist, but since you use `Dir["#{integrations_dir}/*"]` we should be 🆗 .